importers/djornl/parser.py

```diff
@@ -77,34 +153,56 @@ def load_edges(self):
         # dict of nodes, indexed by node ID (node1 and node2 from the file)
         node_ix = {}
         edges = []
-        node_name = self.config('_NODE_NAME')
-        expected_col_count = self.config('_EDGE_FILE_COL_COUNT')
+        node_name = self.config('node_name')
+
+        def edge_type(row):
+            if row['layer_descrip'] not in edge_type_remap:
+                raise RuntimeError(
+                    f"{file['path']} line {line_no}: invalid edge type: {row['layer_descrip']}"
+                )
+            return edge_type_remap[row['layer_descrip']]
+
+        def _key(row):
+            return '__'.join([
+                row['node1'],
+                row['node2'],
+                edge_type(row),
+                row['edge'],
+            ])
```

Comment:
unique key required by the djornl dataset because the default `_key` is derived from `_to` and `_from`, which is not unique enough